pipeline/splicing/calcVarPriors.py

```diff
@@ -733,65 +763,160 @@ def getMaxMaxEntScanScoreSlidingWindowSNS(variant, windowSize=9, exonicPortionSi
         varPos -= 1
         windowStart += 1
         windowEnd += 1
-        
+
+    return {"windowSeqs": windowSeqs,
+            "windowScores": windowScores,
+            "windowAltMaxEntScanScores": windowAltMaxEntScanScores}
+
+def getMaxMaxEntScanScoreSlidingWindowSNS(variant, exonicPortionSize, deNovoLength, donor=True, deNovo=False, accDonor=False):
+    '''
+    Given a variant, determines the maximum alt MaxEntScan score in 
+       a sliding window of size STD_DONOR_SIZE with the variant in each position (1-STD_DONOR_SIZE) if donor = True
+       a sliding window of size STD_ACC_SIZE with the variant in each position (1-STD_ACC_SIZE) if donor = False
+    If deNovo = True, then function determines score considering de novo splice acceptors
+    If doNovo = False, then de novo acceptors are not considered
```

Comment:
It does consider both de novo donors and de novo acceptors. I realized the deNovo argument was misleading and unnecessary because the donor argument can capture the relevant information. I have deleted it.